url-dsl/src/main/scala/urldsl/language/PathSegment.scala

```diff
@@ -169,19 +193,21 @@ trait PathSegment[T, +A] extends UrlPart[T, A] {
     (_: Unit) => createSegments(default)
   )
 
+  final def ignoreRemaining: PathSegment[T, A] = this / PathSegment.remainingSegments.ignore(Nil)
```

Comment:
This method seems a bit out of place / redundant, no? The other non-symbolic methods here (like `ignore`) are about modifying the handling of the current segment or path, whereas `def ignoreRemaining` just appends another segment – but we already have `/` syntax for that.